src/output.cpp

```diff
@@ -431,8 +425,10 @@ std::string Output::value_to_str(BPFtrace &bpftrace,
     case Type::timestamp_mode: {
```

Comment:
Looking good overall! One last small thing. Can you add `case Type::pointer` and `case Type::reference` right above `timestamp_mode` so the switch is exhaustive and will log a bug if anything tries to call `value_to_str` with these two types on the base class?